libsolidity/ast/Types.cpp

```diff
@@ -616,7 +615,7 @@ TypeResult IntegerType::unaryOperatorResult(Token _operator) const
 			_operator == Token::BitNot)
 		return shared_from_this();
 	else
-		return TypePointer();
+		return TypeResult(TypePointer(), "Allowed unary operators on integers are: delete, +, -, ++ and --.");
```

Comment:
I hope we removed `+` and this is just not rebased....

Reply:
The usage of unary ``+`` is caught by the syntax checker, so that's safe! The question is, if we want to move the error to the type checker or leave it and clean up ``unaryOperatorResult`` for all types (if necessary). I will create a PR :)